hazelcast/src/main/java/com/hazelcast/client/impl/protocol/map/MapMessageType.java

```diff
@@ -0,0 +1,29 @@
+package com.hazelcast.client.impl.protocol.map;
+
+import com.hazelcast.client.impl.protocol.ClientMessageType;
+
+/**
+ * Map message types
+ */
+public enum MapMessageType {
+
+    /**
+     * Map put
+     */
+    MAP_PUT(9),
```

Comment:
Can i assume some enum instances with `id` smaller than `9` ? If that is the case, it seems this `MapMessageType` class is following an id-sequence but this class has no information/relation of those classes which have previous id-sequences. This is increasing fragility.